WordPress/src/main/java/org/wordpress/android/ui/stats/StatsVisitorsAndViewsFragment.java

```diff
@@ -517,7 +526,23 @@ protected void updateUI() {
         mGraphView.highlightBar(barSelectedOnGraph);
     }
 
-    private int getDefaultBarIndex(final VisitModel[] dataToShowOnGraph){
+    private String[] makeAccessibleHorizontalLabels(String[] horizontalLabels) {
+        String[] accessibleLabels = new String[horizontalLabels.length];
+
+        for (int i = 0; i < horizontalLabels.length; i++) {
+            if (getTimeframe() == StatsTimeframe.MONTH) {
+                accessibleLabels[i] = StatsUtils.parseDate(horizontalLabels[i], "MMM", "MMMM");
+            } else if (getTimeframe() == StatsTimeframe.WEEK) {
+                accessibleLabels[i] = getString(R.string.stats_bar_week_desc, horizontalLabels[i]);
```

Comment:
Switched to more verbose format. TalkBack is also not so smart in English :) The reason I format `MMM` to `MMMM`  is because TalkBack has trouble figuring out that MMM is a month without a day at the end.